static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -939,8 +938,26 @@ const QueryField = styled(Field)`
 const BuildSteps = styled(List)`
   gap: ${space(4)};
   max-width: 100%;
+`;
+
+const Body = styled(Layout.Body)`
+  gap: 0 !important;
```

Comment:
I feel like I shouldn't have to use `!important`, but the component isn't responding to that styling if I leave it out

Reply:
an alternative is 

```suggestion
 && {
 gap: 0;
 }
```